distage/distage-core/src/main/scala/izumi/distage/effect/modules/MonixBIODIEffectModule.scala

```diff
@@ -0,0 +1,37 @@
+package izumi.distage.effect.modules
+
+import cats.Parallel
+import cats.effect.{ConcurrentEffect, ContextShift, Timer}
```

Comment:
Is it OK that this file is identical to `MonixDIEffectModule.scala`?

Reply:
@VladPodilnyk this is for `monix.bio.IO` from `monix-bio`, while that one is for `monix.eval.Task` from `monix`. These are for different libraries.

Reply:
Oh, yeah, sorry, I've not spotted that imports are different.